scripts/prepare-release.sh

```diff
@@ -301,10 +301,10 @@ if [[ -n "$SERVICE_AFFECTING" ]]; then
  set_csv_service_affecting_field "$SERVICE_AFFECTING"
 fi
 
-# The following is disabled to unblock rc1 cut of 1.13.0 - it should be renabled before final release.
-if [[ "${OLM_TYPE}" == "managed-api-service" ]]; then
- set_related_images
-fi
+# The following is disabled to unblock 1.21.0 - it should be re-enabled after release.
```

Comment:
I think this will cause related_images to not be present in the new CSV. Although I think we have the manifests geneated for the next release so we could address this later.